source/modulo_components/test/cpp/test_component_interface.cpp

```diff
@@ -0,0 +1,65 @@
+#include <gtest/gtest.h>
+#include <rclcpp/node.hpp>
+#include <rclcpp/publisher.hpp>
+#include <rclcpp/node_options.hpp>
+
+#include "modulo_components/ComponentInterface.h"
+#include "modulo_new_core/EncodedState.hpp"
+
+namespace modulo_components {
+class TestComponentInterface : public ::testing::Test {
```

Comment:
Test fixture classes should follow the naming convention:
```suggestion
class ComponentInterfaceTest : public ::testing::Test {
```